bin/client/opts.rs

```diff
@@ -42,6 +42,16 @@ pub struct Opts {
     )]
     pub token_file: Option<PathBuf>,
 
+    #[clap(
+        long = "http-user",
+        about = "Username for HTTP Basic auth",
+        requires = "http-password"
+    )]
+    pub http_user: Option<String>,
+
+    #[clap(long = "http-password", about = "Password for HTTP Basic auth")]
```

Comment:
This should require `http-user` as well. Also, I'd recommend setting `env` on both of these so the password isn't passed on the command line

Reply:
fixed